worldedit_commands/init.lua

```diff
@@ -516,9 +516,25 @@ minetest.register_chatcommand("/cylinder", {
 	end, check_cylinder),
 })
 
-minetest.register_chatcommand("/pyramid", {
+local check_pyramid = function(name, param)
+	if worldedit.pos1[name] == nil then
+		worldedit.player_notify(name, "no position 1 selected")
+		return nil
+	end
+	local found, _, axis, height, nodename = param:find("^([xyz%?])%s+([+-]?%d+)%s+(.+)$")
+	if found == nil then
+		worldedit.player_notify(name, "invalid usage: " .. param)

```

Comment:
Same comment as before : there are 22 messages starting with "invalid usage" and 6 with "no position" in the file and no a single one is capitalize.
Should I capitalize all of them ?
On top, this code has only been moved to make it common between pyramid and hollow pyramid, but I did not touch the message